### ISSUE TITLE
VoiceControl on iPhone 11 can not use part of the search form

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#6903]
From an accessibility audit regarding the search form:

> There is nothing visible on each drop
> down under both the Published Before and After areas. This makes it impossible
> for some VoiceControl users to activate, because there is no ‘term to speak’. The
> VoiceControl doesn’t know the drop down’s ‘name’ so it can’t open them. This
> should be remediated, so the name of each drop down, such as ‘Year’, ‘Month’,
> ‘Day’ is added so it is visible to all viewers. This should then be in the label
> behind the code, so that it matches and the VoiceControl will then be able to
> activate these drop downs and use this feature.

**Comment by israelcefrin** [comment]
Closing since it is duplicated: https://github.com/pkp/pkp-lib/issues/13307
